File: oltp/src/main/java/ar/edu/uca/oltp/repositories/MateriaRepository.java
package ar.edu.uca.oltp.repositories;

import javax.transaction.Transactional;

import ar.edu.uca.oltp.entities.Materia;
import org.springframework.data.repository.CrudRepository;



@Transactional
public interface MateriaRepository extends CrudRepository<Materia, Integer>{

}

File: oltp/src/main/java/ar/edu/uca/oltp/repositories/MateriaRepository.js
package ar.edu.uca.oltp.repositories;

import org.springframework.data.mongodb.repository.MongoRepository;
import org.springframework.stereotype.Repository;

import ar.edu.uca.oltp.entities.Materia;

@Repository
public interface MateriaRepository extends MongoRepository<Materia, String> {

}
